firebase.go

```diff
@@ -128,6 +129,15 @@ func (a *App) Messaging(ctx context.Context) (*messaging.Client, error) {
 	return messaging.NewClient(ctx, conf)
 }
 
+// AppCheck returns an instance of appcheck.Client.
+func (a *App) AppCheck(ctx context.Context) (*appcheck.Client, error) {
+	conf := &internal.AppCheckConfig{
+		ProjectID: a.projectID,
+		JWKSUrl:   appcheck.JWKSUrl,
```

Comment:
Since `JWKSUrl` is defined inside `appcheck` do we need to pass this through the Config? I think we should be able to directly access `JWKSUrl` in the Client, right?

Reply:
Done.  This was mainly done to make testing cleaner.